### PR TITLE
chore: setup default backoff for tranlate method

### PR DESCRIPTION
**Pull request by STRRL**
sometimes the translating progress would be interrupted by API failure, but after resuming, I get a confusing result about the output #386

This PR append default backoff policy for translation, it would relieve that issue.

### CODE REVIEW

**Comment by yihong0618** [comment]
@STRRL 
nice, will merge after lint

pip install -U black
black .

**Comment by STRRL** [comment]
updated! PTAL @yihong0618

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 -e .
 mkdocs
-mkdocs-material
\ No newline at end of file
+mkdocs-material
+backoff
```
Comment (yihong0618) [inline requirements.txt:4]: you also need add this in setup.py

**Comment by yihong0618** [comment]
others LGTM

**Comment by STRRL** [comment]
Updated! I have no idea about python package  management and what `setup.py` is, if it requires other changes, please let me know!

**Comment by yihong0618** [comment]
Nice thanks will create a new release

**Pull request merged**